### ISSUE TITLE
Add a filter to the line chart

### ISSUE THREAD

**Issue by mahowa** [mahowa/Covid-19-d3#12]
Right now you can filter by state, and change the data to use the cuumlative data. But theres a ton more we can do with this.

Examples are:
- Select the data you want to show (i.e. turn off the cases and only view deaths or vice versa)
- Date picker (this is its own issue but an example none the less)
- How about showing all the states in one chart

**Comment by hyunjiLeeTech** [comment]
Hi, I would like to work on this issue but when I run your project, I only can see the black screen. Can you please check it?

**Comment by mahowa** [comment]
Yes i would suggest working on this in code sandbox. I wanted to make this code super accessible. I will work on getting this to work anywhere

**Comment by irina-chernik** [comment]
@mahowa Hi, the solution from this article with adding browserlist helped me to start the project https://flaviocopes.com/parcel-regeneratorruntime-not-defined/

**Comment by mahowa** [comment]
Thank you @irina-chernik I created and issue #17 If you want to make a pr. This is an easy pr for HacktoberFest Ill assign it to you unless you say otherwise

**Comment by joquendo** [comment]
Hey @mahowa and @hyunjiLeeTech. I would like to try this out by adding checkboxes at the top to toggle the data between 'cases' and 'deaths'. Is it okay to create PR if I manage to toggle?

Edit: if this is being worked on, maybe I can attempt the Date picker.

**Comment by mahowa** [comment]
@joquendo I think adding checkboxes for cases and deaths would be a great thing to work on

**Comment by joquendo** [comment]
OK. I'm working on it. D3JS is more involved than I initially thought, but I'm understanding it more now. I plan to work on it today and tomorrow. Trying to submit before the end of the hackathon.
